packages/@webex/plugin-meetings/test/unit/spec/meeting/index.js

```diff
@@ -4196,6 +4200,8 @@ describe('plugin-meetings', () => {
             // and that these were the only /media requests that were sent
             assert.calledTwice(locusMediaRequestStub);
             assert.calledOnce(handleDeviceLoggingSpy);
+            assert.calledWith(handleDeviceLoggingSpy, false);
+
           });
 
```

Comment:
Please add test for the following conditions
1. Only asked for camera permission
2. Only asked for mic permission
3. Ask for both permissions
4. Ask for neither permissions.